profiler/example_run_ncu.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+sudo /opt/nvidia/nsight-compute/2021.1.0/ncu -f -o results.nvprof \
+    --target-processes all --import-source on \
+    /home/floriand/venv/vcm_1_0/bin/python3 \
```

Comment:
A general `ncu` has no home yet because you either need to run on GCP and potentially later on NASA's PRISM, since the P100 do not have performance counters. So this example is very local for now (my GCP box). One of my first TODO if we get a solid access to PRISM is to make a .sh for it

Reply:
so should this document then be here in this form (as an example)? Or should it be a template? Or should there just be text in a readme that explains how to do this?

Reply:
I've verbosed the .sh itself, the readme is already pointing to it